base/ltfssaxes.dtx

```diff
@@ -192,503 +249,3176 @@
+\DeclareFontSeriesChangeRule {bx}{el}   {elx}  {}
+\DeclareFontSeriesChangeRule {bx}{l}    {lx}   {}
+\DeclareFontSeriesChangeRule {bx}{sl}   {slx}  {}
+\DeclareFontSeriesChangeRule {bx}{sb}   {sbx}  {bx}
+\DeclareFontSeriesChangeRule {bx}{b}    {bx}   {}
+\DeclareFontSeriesChangeRule {bx}{eb}   {ebx}  {bx}
+\DeclareFontSeriesChangeRule {bx}{ub}   {ubx}  {bx}
+\DeclareFontSeriesChangeRule {bx}{uc}   {buc}  {}
+\DeclareFontSeriesChangeRule {bx}{ec}   {bec}  {}
+\DeclareFontSeriesChangeRule {bx}{c}    {bc}   {}
+\DeclareFontSeriesChangeRule {bx}{sc}   {bsc}  {}
+\DeclareFontSeriesChangeRule {bx}{sx}   {bsx}  {}
+\DeclareFontSeriesChangeRule {bx}{x}    {bx}   {}
+\DeclareFontSeriesChangeRule {bx}{ex}   {bex}  {}
+\DeclareFontSeriesChangeRule {bx}{ux}   {bux}  {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {bex}{ul}  {ulex} {}
+\DeclareFontSeriesChangeRule {bex}{el}  {elex} {}
+\DeclareFontSeriesChangeRule {bex}{l}   {lex}  {}
+\DeclareFontSeriesChangeRule {bex}{sl}  {slex} {}
+\DeclareFontSeriesChangeRule {bex}{sb}  {sbex} {bex}
+\DeclareFontSeriesChangeRule {bex}{b}   {bex}  {}
+\DeclareFontSeriesChangeRule {bex}{eb}  {ebex} {bex}
+\DeclareFontSeriesChangeRule {bex}{ub}  {ubex} {bex}
+\DeclareFontSeriesChangeRule {bex}{uc}  {buc}  {}
+\DeclareFontSeriesChangeRule {bex}{ec}  {bec}  {}
+\DeclareFontSeriesChangeRule {bex}{c}   {bc}   {}
+\DeclareFontSeriesChangeRule {bex}{sc}  {bsc}  {}
+\DeclareFontSeriesChangeRule {bex}{sx}  {bsx}  {}
+\DeclareFontSeriesChangeRule {bex}{x}   {bx}   {}
+\DeclareFontSeriesChangeRule {bex}{ex}  {bex}  {}
+\DeclareFontSeriesChangeRule {bex}{ux}  {bux}  {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {bux}{ul}  {ulux} {}
+\DeclareFontSeriesChangeRule {bux}{el}  {elux} {}
+\DeclareFontSeriesChangeRule {bux}{l}   {lux}  {}
+\DeclareFontSeriesChangeRule {bux}{sl}  {slux} {}
+\DeclareFontSeriesChangeRule {bux}{sb}  {sbux} {bux}
+\DeclareFontSeriesChangeRule {bux}{b}   {bux}  {}
+\DeclareFontSeriesChangeRule {bux}{eb}  {ebux} {bux}
+\DeclareFontSeriesChangeRule {bux}{ub}  {ubux} {bux}
+\DeclareFontSeriesChangeRule {bux}{uc}  {buc}  {}
+\DeclareFontSeriesChangeRule {bux}{ec}  {bec}  {}
+\DeclareFontSeriesChangeRule {bux}{c}   {bc}   {}
+\DeclareFontSeriesChangeRule {bux}{sc}  {bsc}  {}
+\DeclareFontSeriesChangeRule {bux}{sx}  {bsx}  {}
+\DeclareFontSeriesChangeRule {bux}{x}   {bx}   {}
+\DeclareFontSeriesChangeRule {bux}{ex}  {bex}  {}
+\DeclareFontSeriesChangeRule {bux}{ux}  {bux}  {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {ebuc}{ul} {uluc} {}
+\DeclareFontSeriesChangeRule {ebuc}{el} {eluc} {}
+\DeclareFontSeriesChangeRule {ebuc}{l}  {luc}  {}
+\DeclareFontSeriesChangeRule {ebuc}{sl} {sluc} {}
+\DeclareFontSeriesChangeRule {ebuc}{sb} {sbuc} {buc}
+\DeclareFontSeriesChangeRule {ebuc}{b}  {buc}  {}
+\DeclareFontSeriesChangeRule {ebuc}{eb} {ebuc} {}
+\DeclareFontSeriesChangeRule {ebuc}{ub} {ubuc} {buc}
```

Comment:
```suggestion
\DeclareFontSeriesChangeRule {ebuc}{ub} {ubuc} {ebuc}
```

This one change somehow didn't make it into https://github.com/latex3/latex2e/pull/1583/commits/71e5638d8ca7c82cd8a51b56c10e9ac67f0ae21c.